demo/index.html

```diff
@@ -198,12 +203,11 @@
 
     // event handlers
     mgcComponent.addEventListener('EmailReceiptRequest', (e) => {
-      const uxMessageInfoArea = document.getElementById('interaction-status-area');
 
       const { donation } = e.detail;
-      const randomizer = Math.floor(Math.random() + 0.5);
-      const successOrFail = randomizer === 1 ? 'success' : 'fail';
-      const returnTiming = randomizer === 1 ? 3000 : 8000;
+      const heads = coinFlip() === 1;
+      const successOrFail = heads ? 'success' : 'fail';
+      const returnTiming = heads ? 3000 : 8000;
```

Comment:
QUESTION: What's the significance of the randomization of this timing? Should the choice of timing be displayed somewhere in a dev UI so the developer knows which one was used?

Reply:
In the demo, this mocks a request so the component feedback loop isn't immediate.